car-service-deployment.yaml

```diff
@@ -0,0 +1,40 @@
+apiVersion: apps/v1
+kind: Deployment
+metadata:
+  annotations:
+    kompose.cmd: C:\Windows\system32\kompose.exe convert
```

Comment:
Well, what is it?

Any related system code should not be appear in git

Reply:
Remove